packages/SwingSet/test/test-network.js

```diff
@@ -1,7 +1,8 @@
-// @ts-check
+/* global harden */
+import '@agoric/install-ses'; // adds 'harden' to global
 import { test } from 'tape-promise/tape';
 import { producePromise } from '@agoric/produce-promise';
-import rawHarden from '@agoric/harden';
+// import rawHarden from '@agoric/harden';
```

Comment:
Might be able to just delete this line.

Reply:
Oops, that was left over from a typescript experiment that I broke and swept the pieces under the rug before the author could notice. Missed a fragment :).